libs/community/tests/integration_tests/tools/zenguard/test_zenguard.py

```diff
@@ -0,0 +1,105 @@
+import os
+from typing import Any, Dict, List
+
+import pytest
+
+from langchain_community.tools.zenguard.tools import Detector, ZenGuardTool
+
+
+@pytest.fixture()
+def zenguard_tool() -> ZenGuardTool:
+    zenguard_api_key = os.getenv("ZENGUARD_API_KEY")
+    if zenguard_api_key is None:
+        zenguard_api_key = "3Ev_DGvELv7EnlgWMTlpmWTo82tpstyz4Li_R7kTDQw"    
```

Comment:
```suggestion
```
It looks like maintainers aren't allowed to modify this branch but I'd remove this and consider the API key compromised.

Reply:
yup, this API key is blacklisted from the very beginning :)